renderers/games.py

```diff
@@ -64,9 +65,10 @@ def render(self):
 
       # TODO: https://github.com/ajbowler/mlb-led-scoreboard/issues/30
       # The time_delta comparison will need to change depending on scrolling text size
-      if self.config.rotate_games and time_delta >= FIFTEEN_SECONDS:
+      if self.config.rotate_games and time_delta >= FIFTEEN_SECONDS and self.current_scrolling_finished_once:

```

Comment:
Unless I'm missing something this will cause games in progress to never rotate. You'd need to also verify that the game is in a pregame status when checking if the scrolling has finished.